### PR TITLE
Update sklearn_spacy_text_classifier_example.ipynb

### LINKED ISSUE #2486: Unreadable notebook - sklearn_spacy_text_classifier_example.ipynb
## Describe the bug

Missing `",` at line 460 for [`sklearn_spacy_text_classifier_example.ipynb`](../blob/master/examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb)

## To reproduce

Go to the link above

## Expected behaviour

Readable notebook

## Environment

GitHub

## Model Details 
NA

### PR DESCRIPTION
**Pull request by Syakyr**
Missing `",` at line 460, rendered notebook unusable without it

**What this PR does / why we need it**:
Added `",` at line 460 for the specified notebook, otherwise notebook is rendered unusable.

**Which issue(s) this PR fixes**:

Fixes #2486 

**Special notes for your reviewer**:
NA

**Does this PR introduce a user-facing change?**:

```release-note
Fixes sklearn_spacy_text_classifier_example.ipynb to be readable
```

### CODE REVIEW

**Comment by seldondev** [comment]
Hi @Syakyr. Thanks for your PR.

I'm waiting for a [SeldonIO](https://github.com/orgs/SeldonIO/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/SeldonIO/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **axsaucedo**
You can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by RafalSkolasinski** [comment]
nice catch, /approve

**Comment by seldondev** [comment]
Mon Sep 28 12:17:24 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2487/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2487 --build=2

**Review by RafalSkolasinski — APPROVED** [review]
/lgtm

**Comment by seldondev** [comment]
Mon Sep 28 12:18:00 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2487/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2487 --build=1

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[RafalSkolasinski](https://github.com/SeldonIO/seldon-core/pull/2487#issuecomment-700606135 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com#issuecomment-700606135/SeldonIO/seldon-core/blob/master/OWNERS)~~ [RafalSkolasinski]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Tue Sep 29 10:14:40 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2487/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2487 --build=3

**Comment by seldondev** [comment]
Tue Sep 29 10:15:59 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2487/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2487 --build=4

**Pull request merged**
